src/Umbraco.Core/Services/IRelationService.cs

```diff
@@ -4,6 +4,17 @@
 
 namespace Umbraco.Cms.Core.Services;
 
+/// <summary>
+/// Definition of relation directions used as a filter when requesting if a given item has relations.
+/// </summary>
+[Flags]
+public enum RelationDirectionFilter
+{
+    Parent = 1,
+    Child = 2,
+    Any = Parent | Child
+}
+
```

Comment:
I think this should be in `Core.Models` instead of the same file as the interface since it's used outside the interface itself too :smile: